src/example-relay/src/components/Navbar.js

```diff
@@ -106,6 +106,9 @@ function Navbar() {
 }
 
 const styles = sx.create({
+  notUsing: {
```

Comment:
@mrtnzlml shouldn't this produce a lint error?

Reply:
It should. I am going to check what's wrong.

Reply:
Should be fixed here: https://github.com/adeira/universe/pull/1324